src/validator.ts

```diff
@@ -188,49 +193,261 @@ export const possiblyReturningBlock = warning<Method>(node => {
 })
 
 export const shouldNotUseOverride = error<Method>(node =>
-  !node.isOverride || !!node.parent().lookupMethod(node.name, node.parameters.length, { lookupStartFQN: node.parent().fullyQualifiedName(), allowAbstractMethods: true })
+  !node.isOverride || !!superclassMethod(node)
+)
+
+export const namedArgumentShouldExist = error<NamedArgument>(node => {
+  const parent = getReferencedModule(node.parent())
+  return !!parent && parent.hasField(node.name)
+})
+
+export const namedArgumentShouldNotAppearMoreThanOnce = warning<NamedArgument>(node =>  {
+  const nodeParent = node.parent()
+  let siblingArguments: List<NamedArgument> | undefined
+  if (nodeParent.is('New')) siblingArguments = nodeParent.args
+  return !siblingArguments || siblingArguments.filter(_ => _.name === node.name).length === 1
```

Comment:
Veo que usamos mucho `filter(f).length` . No hace falta hacerlo ahora, pero capaz podemos poner un TODO para crear un extension `count(f)`

Reply:
**count** me in!